taxi/forms.py

```diff
@@ -1,35 +1,52 @@
 from django import forms
 from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
 from django.core.exceptions import ValidationError
 
 from taxi.models import Driver
 
 
-class DriverLicenseUpdateForm(forms.ModelForm):
-    LICENSE_LENGTH = 8
+class DriverCreationForm(UserCreationForm):
+    class Meta:
+        model = Driver
+        fields = UserCreationForm.Meta.fields + (
+            "username",
+            # "first_name",
+            # "last_name",

```

Comment:
you've probably forget to remove comments here? i guess first name and last name should be added